Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/VmServiceWrapper.java

```diff
@@ -116,6 +117,11 @@ public void received(final Isolate isolate) {
                         return;
                       }
 
+                      // This is the entry point for attaching a debugger to a running app.
+                      if (eventKind == EventKind.Resume) {
```

Comment:
Do you know how to reach this branch?

Reply:
I reverted the changes in this file.  When I saw the difference to what the Dart Plugin (as compared to the Flutter plugin) I originally thought that the changes were for webdev.  @stevemessick advice as necessary.